doc/platform_api/CMIS_Diagnostic_Monitoring_Overview_in_SONiC.md

```diff
@@ -0,0 +1,1685 @@
+```
+
+#### 2.1.4 Transceiver DOM flag change count data
+
+The `TRANSCEIVER_DOM_FLAG_CHANGE_COUNT` table stores the flag change count for the DOM flags.
+
+lane_num: Represents lane number of the field. The lane number is an integer value that ranges from 1 to 8.
+
+```plaintext
+    ;Defines Transceiver DOM flag change count for a port
+    key                              = TRANSCEIVER_DOM_FLAG_CHANGE|ifname   ; information module DOM flags change count on port
+    ; field                          = value
+    temphighalarm_chg_cnt                 = INTEGER           ; temperature high alarm change count
+    temphighwarning_chg_cnt               = INTEGER           ; temperature high warning change count
+    templowalarm_chg_cnt                  = INTEGER           ; temperature low alarm change count
+    templowwarning_chg_cnt                = INTEGER           ; temperature low warning change count
+    vcchighalarm_chg_cnt                  = INTEGER           ; vcc high alarm change count
+    vcchighwarning_chg_cnt                = INTEGER           ; vcc high warning change count
+    vcclowalarm_chg_cnt                   = INTEGER           ; vcc low alarm change count
+    vcclowwarning_chg_cnt                 = INTEGER           ; vcc low warning change count
+    txpowerhighalarm_chg_cnt{lane_num}    = INTEGER           ; tx power high alarm change count
+    txpowerlowalarm_chg_cnt{lane_num}     = INTEGER           ; tx power low alarm change count
+    txpowerhighwarning_chg_cnt{lane_num}  = INTEGER           ; tx power high warning change count
+    txpowerlowwarning_chg_cnt{lane_num}   = INTEGER           ; tx power low alarm change count
+    rxpowerhighalarm_chg_cnt{lane_num}    = INTEGER           ; rx power high alarm change count
+    rxpowerlowalarm_chg_cnt{lane_num}     = INTEGER           ; rx power low alarm change count
+    rxpowerhighwarning_chg_cnt{lane_num}  = INTEGER           ; rx power high warning change count
+    rxpowerlowwarning_chg_cnt{lane_num}   = INTEGER           ; rx power low warning change count
+    txbiashighalarm_chg_cnt{lane_num}     = INTEGER           ; tx bias high alarm change count
+    txbiaslowalarm_chg_cnt{lane_num}      = INTEGER           ; tx bias low alarm change count
+    txbiashighwarning_chg_cnt{lane_num}   = INTEGER           ; tx bias high warning change count
+    txbiaslowwarning_chg_cnt{lane_num}    = INTEGER           ; tx bias low warning change count
+    lasertemphighalarm_chg_cnt            = INTEGER           ; laser temperature high alarm change count
+    lasertemplowalarm_chg_cnt             = INTEGER           ; laser temperature low alarm change count
+    lasertemphighwarning_chg_cnt          = INTEGER           ; laser temperature high warning change count
+    lasertemplowwarning_chg_cnt           = INTEGER           ; laser temperature low warning change count
+```
+
+#### 2.1.5 Transceiver DOM flag time set data
```

Comment:
if all the data type is in string, is it easy to convert it to datetime data type? In some cases we may need to parse this data and do time diff.

Reply:
Always show the current time from any CLI output would help us determining the time difference too